### ISSUE TITLE
Datatable multisort broken

### ISSUE THREAD

**Issue by Mobe91** [primefaces/primereact#617]
### There is no guarantee in receiving an immediate response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeReact PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[ x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://forum.primefaces.org/viewforum.php?f=57
```

**Current behavior**
Among other issues, the toggling of sort order in multiSortMeta does not work.

**Expected behavior**
Multisort works.

* **React version:**
16.5.2

* **PrimeReact version:**
2.0.0-beta.9

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
all
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
all

**Comment by GrubySG** [comment]
Hello. Still not working in 3.2.0. I can't change sort order when using multiSortMeta.
Example:
https://codesandbox.io/s/primereact-test-95kog

**Comment by mertsincan** [comment]
Thanks a lot for the sample. I'll check and get back to you.

**Comment by Mobe91** [comment]
@SebastianMila You might want to try my PR #620

**Comment by mertsincan** [comment]
Hi @Mobe91,

Thanks a lot for the PR! I made some changes using your PR ;)

Thanks again,
Best Regards,

**Comment by Mobe91** [comment]
@mertsincan This is still not merged in 4.0.0. Can you reopen this issue please?
